InTheHand.BluetoothLE/GattCharacteristicUuids.cs

```diff
@@ -415,9 +433,83 @@ public static BluetoothUuid FromBluetoothUti(string bluetoothUti)
+        public static readonly BluetoothUuid IddRecordAccessControlPoint = 0x2B27;
+        
+        [BluetoothUti("org.bluetooth.characteristic.idd_status")]
+        public static readonly BluetoothUuid IddStatus = 0x2B21;
+        
+        [BluetoothUti("org.bluetooth.characteristic.idd_status_changed")]
+        public static readonly BluetoothUuid IddStatusChanged = 0x2B20;
+        
+        [BluetoothUti("org.bluetooth.characteristic.idd_status_reader_control_point")]
+        public static readonly BluetoothUuid IddStatusReaderControlPoint = 0x2B24;
+   
+        [BluetoothUti("org.bluetooth.characteristic.ieee_11073-20601_regulatoryertification_data_list")]
+        public static readonly BluetoothUuid RegulatoryCertificationDataList = 0x2A2A;
+   
+        [BluetoothUti("org.bluetooth.characteristic.indoor_bike_data")]
+        public static readonly BluetoothUuid IndoorBikeData = 0x2AD2;
+
+        [BluetoothUti("org.bluetooth.characteristic.indoor_positioning_configuration")]
+        public static readonly BluetoothUuid IndoorPositioningConfiguration = 0x2AAD;
+        
+        [BluetoothUti("org.bluetooth.characteristic.intermediate_cuff_pressure")]
+        public static readonly BluetoothUuid IntermediateCuffPressure = 0x2A36;
+        
+        [BluetoothUti("org.bluetooth.characteristic.intermediate_temperature")]
+        public static readonly BluetoothUuid IntermediateTemperature = 0x2A1E;
+        
+        [BluetoothUti("org.bluetooth.characteristic.irradiance")]
+        public static readonly BluetoothUuid Irradiance = 0x2A77;
+        
+        [BluetoothUti("org.bluetooth.characteristic.language")]
+        public static readonly BluetoothUuid Language = 0x2AA2;
+        
+        [BluetoothUti("org.bluetooth.characteristic.last_name")]
+        public static readonly BluetoothUuid LastName = 0x2A90;
+        
+        [BluetoothUti("org.bluetooth.characteristic.latitude")]
+        public static readonly BluetoothUuid Latitude = 0x2AAE;
+       
+        [BluetoothUti("org.bluetooth.characteristic.ln_control_point")]
+        public static readonly BluetoothUuid LNControlPoint = 0x2A6B;
+        
+        [BluetoothUti("org.bluetooth.characteristic.ln_feature")]
+        public static readonly BluetoothUuid LNFeature = 0x2A6A;
+        
+        [BluetoothUti("org.bluetooth.characteristic.local_east_coordinate")]
+        public static readonly BluetoothUuid LocalEastCoordinate = 0x2AB1;
+        
+        [BluetoothUti("org.bluetooth.characteristic.local_north_coordinate")]
+        public static readonly BluetoothUuid LocalNorthCoordinate = 0x2AB0;
+        
+        [BluetoothUti("org.bluetooth.characteristic.local_time_information")]
+        public static readonly BluetoothUuid LocalTimeInformation = 0x2A0F;
+        
+        [BluetoothUti("org.bluetooth.characteristic.location_and_speed")]
+        public static readonly BluetoothUuid LocationandSpeedCharacteristic = 0x2A67;
```

Comment:
Although it is in the Assigned Numbers document I don't think we should include Characteristic in the name as these are all Characteristic UUIDs. Please change to "LocationAndSpeed"

Reply:
yes, that makes sense.